modules/ROOT/pages/update-acm.adoc

```diff
@@ -14,6 +14,22 @@ Because API Community Manager follows the standard support policy for MuleSoft c
 
 For information about upgrade notifications, see xref:notifications.adoc[important notifications].
 
+== Upgrading to Version 2.7.0
+
+Instead of using user profiles, API Community Manager now uses the following permission sets to define the visibility and access to external objects and Apex classes. 
```

Comment:
```suggestion
Instead of using user profiles, API Community Manager now uses the following permission sets to define the visibility and access to external objects and Apex classes: 
```
```suggestion
Instead of using user profiles, API Community Manager now uses the following permission sets to define the visibility and access to external objects and Apex classes. 
```